harness/determined/layers/_workload_manager.py

```diff
@@ -208,6 +208,15 @@ def _respond(in_response: workload.Response) -> None:
                 )
                 sys.exit(1)
```

Comment:
You're so close to this dead sys.exit()... you just just delete it while you are here.

Reply:
also the one lower down in this file.

Reply:
I had an itch to, but honestly I thought you were more against that sort of thing. No objections on my end!